<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAQ_GUI_QTClass</class>
 <widget class="QMainWindow" name="DAQ_GUI_QTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DAQ_GUI_QT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>661</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1029</width>
       <height>659</height>
      </rect>
     </property>
     <widget class="QtCharts::QChartView" name="chartView_0">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>571</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>540</y>
        <width>160</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>开门时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_time_open_door">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>关门时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_time_close_door_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>当前时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_current_time">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>61</width>
        <height>44</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>采集次数:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_count">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>130</y>
        <width>259</width>
        <height>401</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_enable">
           <property name="styleSheet">
            <string notr="true">min-width:     30px;    
min-height:    30px;    
max-width:     30px;   
max-height:    30px;   
border-radius: 16px;      
background-color: rgb(175, 176, 179);
border-style: solid;
border-width: 2px;
border-color: rgb(131, 132, 135);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_19">
           <property name="font">
            <font>
             <family>等线</family>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>零速信号</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_21">
         <item>
          <widget class="QLabel" name="label_open">
           <property name="styleSheet">
            <string notr="true">min-width:     30px;    
min-height:    30px;    
max-width:     30px;   
max-height:    30px;   
border-radius: 16px;      
background-color: rgb(175, 176, 179);
border-style: solid;
border-width: 2px;
border-color: rgb(131, 132, 135);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_20">
           <property name="font">
            <font>
             <family>等线</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>集控开门</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_22">
         <item>
          <widget class="QLabel" name="label_close">
           <property name="styleSheet">
            <string notr="true">min-width:     30px;    
min-height:    30px;    
max-width:     30px;   
max-height:    30px;   
border-radius: 16px;      
background-color: rgb(175, 176, 179);
border-style: solid;
border-width: 2px;
border-color: rgb(131, 132, 135);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_21">
           <property name="font">
            <font>
             <family>等线</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>集控关门</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_23">
         <item>
          <widget class="QLabel" name="label_door_close_1">
           <property name="styleSheet">
            <string notr="true">min-width:     30px;    
min-height:    30px;    
max-width:     30px;   
max-height:    30px;   
border-radius: 16px;      
background-color: rgb(175, 176, 179);
border-style: solid;
border-width: 2px;
border-color: rgb(131, 132, 135);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_22">
           <property name="font">
            <font>
             <family>等线</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>门关闭1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_26">
         <item>
          <widget class="QLabel" name="label_door_close_2">
           <property name="styleSheet">
            <string notr="true">min-width:     30px;    
min-height:    30px;    
max-width:     30px;   
max-height:    30px;   
border-radius: 16px;      
background-color: rgb(175, 176, 179);
border-style: solid;
border-width: 2px;
border-color: rgb(131, 132, 135);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_23">
           <property name="font">
            <font>
             <family>等线</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>门关闭2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_27">
         <item>
          <widget class="QLabel" name="label_lock">
           <property name="styleSheet">
            <string notr="true">min-width:     30px;    
min-height:    30px;    
max-width:     30px;   
max-height:    30px;   
border-radius: 16px;      
background-color: rgb(175, 176, 179);
border-style: solid;
border-width: 2px;
border-color: rgb(131, 132, 135);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_24">
           <property name="font">
            <font>
             <family>等线</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>锁到位</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_29">
         <item>
          <widget class="QLabel" name="label_v_signal">
           <property name="styleSheet">
            <string notr="true">min-width:     30px;    
min-height:    30px;    
max-width:     30px;   
max-height:    30px;   
border-radius: 16px;      
background-color: rgb(175, 176, 179);
border-style: solid;
border-width: 2px;
border-color: rgb(131, 132, 135);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_26">
           <property name="font">
            <font>
             <family>等线</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>电源电压</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_28">
         <item>
          <widget class="QLabel" name="label_v_signal_posSen">
           <property name="styleSheet">
            <string notr="true">min-width:     30px;    
min-height:    30px;    
max-width:     30px;   
max-height:    30px;   
border-radius: 16px;      
background-color: rgb(175, 176, 179);
border-style: solid;
border-width: 2px;
border-color: rgb(131, 132, 135);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_25">
           <property name="font">
            <font>
             <family>等线</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>位置传感器电源电压信号</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_start_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>13</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>265</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_24">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>当前连接到电脑的采集卡数量：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_daq_count">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_refresh">
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_start">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>340</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>160</width>
        <height>45</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>通道模式:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_chan_mode">
         <item>
          <property name="text">
           <string>1.DIFF(差分模式)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2.NRSE(非接地参考单端)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3.RSE(接地参考单端)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_dio_stop">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>当前波形：</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_dio_start">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>开始采集</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_stop">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>360</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>结束</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>240</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_25">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>采集卡ID：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_devices"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_connect_daq">
         <property name="text">
          <string>打开采集卡</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_is_connected">
         <property name="styleSheet">
          <string notr="true">min-width:     14px;    
min-height:    14px;    
max-width:     14px;   
max-height:    14px;   
border-radius: 8px;      
background-color: rgb(234, 70, 48);
border-style: solid;
border-width: 1px;
border-color: rgb(131, 132, 135);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>151</width>
        <height>142</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QCheckBox" name="checkBox_all">
           <property name="text">
            <string>全选</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>通道注释</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QCheckBox" name="checkBox_ai_0">
           <property name="text">
            <string>AI 0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_4">
           <property name="text">
            <string>主轴阻塞制动器</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 128, 24);
min-width:     10px;    
min-height:    10px;    
max-width:     10px;   
max-height:    10px; 
border-style: solid;
border-width: 1px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QCheckBox" name="checkBox_ai_1">
           <property name="text">
            <string>AI 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_5">
           <property name="text">
            <string>电磁制动器（解锁）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 249);
min-width:     10px;    
min-height:    10px;    
max-width:     10px;   
max-height:    10px; 
border-style: solid;
border-width: 1px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <widget class="QCheckBox" name="checkBox_ai_2">
           <property name="text">
            <string>AI 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_16">
           <property name="text">
            <string>驱动电机信号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(134, 0, 125);
min-width:     10px;    
min-height:    10px;    
max-width:     10px;   
max-height:    10px; 
border-style: solid;
border-width: 1px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="checkBox_ai_3">
           <property name="text">
            <string>AI 3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="text">
            <string>位置传感器通道A/B</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 0, 24);
min-width:     10px;    
min-height:    10px;    
max-width:     10px;   
max-height:    10px; 
border-style: solid;
border-width: 1px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>280</y>
        <width>80</width>
        <height>44</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>采样率(Sps/s):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_sample_rate">
         <property name="text">
          <string>4000</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>66</width>
        <height>45</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>量程:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_range">
         <item>
          <property name="text">
           <string>-5~5V</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0~10V</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选项</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>文件保存路径</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="DAQ_GUI_QT.qrc"/>
 </resources>
 <connections/>
</ui>
